test/acceptance/committee_election_test.go

```diff
@@ -25,6 +26,13 @@ func TestCommitteeElections_OneElectionAndCheckReputationChanges(t *testing.T) {
 		WithManagementPollingInterval(20*time.Millisecond).
 		WithLogFilters(log.DiscardAll()).
 		WithConsensusAlgos(consensus.CONSENSUS_ALGO_TYPE_LEAN_HELIX).
```

Comment:
i don't like this way of over-riding config ... there is config for acceptance ... this way of override i want to kill.